_includes/cockroachcloud/planning-your-serverless-cluster.md

```diff
@@ -0,0 +1,11 @@
+### Resource usage
+
+All resource usage in CockroachCloud Serverless (beta) is measured in Request Units, or RUs. RUs represent the compute and I/O resources used by a read or a write query. All database operations cost a certain amount of RUs depending on the resources used. For example, a "small read" might cost 1 RU, and a "large read" such as a full table scan with indexes could cost 100 RUs. You can see how many request units your cluster has used on the [Cluster Overview](#view-cluster-overview) page.
+
+### Cluster scaling
+
+CockroachCloud Serverless (beta) clusters scale based on your workload. Free clusters include 250M RUs with 10M RUs reserved for [burst performance](architecture.html#concepts) and 5GiB of storage per month. They have a guaranteed baseline performance of 100 RUs per second, or 2.5K QPS.
```

Comment:
2.5K QPS is not true since the smallest query. A more accurate way to describe this dynamic would be to say that you can get "up to 100 QPS."